python/py-sure/Portfile

```diff
@@ -19,31 +19,34 @@ homepage            http://github.com/gabrielfalcao/sure
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           sha256  3c8d5271fb18e2c69e2613af1ad400d8df090f1456081635bd3171847303cdaa \
-                    rmd160  fb6c37da7c2d005c6874d266fbc8d2edcb21d833 \
-                    size    45933
+checksums           sha256  34ae88c846046742ef074036bf311dc90ab152b7bc09c342b281cebf676727a2 \
+                    rmd160  5a20615bf48f287c322c9b17f651ffd8ac680a1b \
+                    size    46747
 
-python.versions     37
+python.versions     37 38 39 310
 
 if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
-    post-extract {
-        system "find ${worksrcpath} -type d -print0 | xargs -0 chmod a+rx"
-        system "find ${worksrcpath} -type f -print0 | xargs -0 chmod a+r"
-    }
+    # See https://github.com/gabrielfalcao/sure/issues/169
+    patchfiles      patch-fix-python310.diff
 
     depends_run-append \
                     port:py${python.version}-mock \
                     port:py${python.version}-six
 
     depends_test-append \
+                    port:py${python.version}-coverage \
                     port:py${python.version}-mock \
                     port:py${python.version}-nose \
+                    port:py${python.version}-rednose \
                     port:py${python.version}-six
 
     test.run        yes
+    test.cmd        ${python.bin}
+    test.target     setup.py test
```

Comment:
this `test.cmd` and `test.target` are the defaults set by the `python` PG - no need to add them here